specification/communication/data-plane/Sms/preview/2024-01-14-preview/communicationservicessms.json

```diff
@@ -0,0 +1,355 @@
+            "$ref": "./examples/MmsSendMessageAsync.json"
+          }
+        }
+      }
+    },
+    "/sms": {
+      "post": {
+        "tags": [
+          "Sms"
+        ],
+        "summary": "Sends a SMS from a phone number that is acquired by the authenticated account.",
+        "operationId": "Sms_Send",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "SendMessageRequest",
+            "description": "",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/SendMessageRequest"
+            }
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted",
+            "schema": {
+              "$ref": "#/definitions/SmsSendResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Send Message": {
+            "$ref": "./examples/SmsSendMessageAsync.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "MmsSendMessageRequest": {
+      "description": "Represents the properties of a send message request.",
+      "required": [
+        "attachments",
+        "from",
+        "recipients"
+      ],
+      "type": "object",
+      "properties": {
+        "from": {
+          "description": "The from identifier that is owned by the authenticated account",
+          "type": "string"
+        },
+        "recipients": {
+          "description": "The to phone numbers in E.164 format.",
+          "maxItems": 100,
+          "minItems": 1,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/MmsSendRecipient"
+          }
+        },
+        "message": {
+          "description": "The contents of the message that will be sent to the recipient.",
+          "maxLength": 1000,
+          "minLength": 0,
+          "type": "string"
+        },
+        "attachments": {
+          "description": "A list of media attachments to include as part of the MMS. You can have maximum 10 attachments.",
+          "maxItems": 10,
+          "minItems": 1,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/MmsSendRequestAttachment"
+          }
+        },
+        "sendOptions": {
+          "$ref": "#/definitions/MmsSendOptions"
+        }
+      }
+    },
+    "MmsSendOptions": {
+      "description": "Used for sending more options in the request",
+      "required": [
+        "enableDeliveryReport"
+      ],
+      "type": "object",
+      "properties": {
+        "enableDeliveryReport": {
+          "description": "Get or Set the notify delivery report",
+          "type": "boolean"
+        },
+        "tag": {
+          "description": "Get or Set the Customer Content\r\nShould not be logged.",
```

Comment:
Let's use the description from the current stable version.

Reply:
ok